File: web3rpc/rpc-specs/code-samples/python/eth/block/getStorageAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x295a70b2de5e3953354a6a8344e616ed314d7251"
quantity = "0x0"
blockTag = "latest"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_storage_at(address, quantity, blockTag)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getStorageAt.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const address = "0x295a70b2de5e3953354a6a8344e616ed314d7251";
  const quantity = "0x0";
  const blockNumberOrHashOrTag = "latest";

  provider.eth
    .getStorageAt(
      address,
      quantity,
      blockNumberOrHashOrTag,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
